r/R/dplyr-funcs-datetime.R

```diff
@@ -189,6 +189,65 @@ register_bindings_datetime <- function() {
   })
 }
 
+register_bindings_duration <- function() {
+  register_binding("difftime", function(time1,
+                                        time2,
+                                        tz,
+                                        units = c("auto", "secs", "mins",
+                                                  "hours", "days", "weeks")) {
+    units <- match.arg(units)
+    if (units != "secs") {
+      abort("`difftime()` with units other than seconds not supported in Arrow")
+    }
```

Comment:
We should probably make "secs" be the default for this argument. I know it's "auto" for base R, but we don't (yet? may never?) have the facility to map from auto to the duration that R would give + we don't want the default arg to always cause a failure.

It might be nice to also mention that the argument name is `"secs"` in the message — I always end up needing to go look up which abbreviation it is.

Reply:
Done